test/sequential/rolldown.withdraw.fees.test.ts

```diff
@@ -0,0 +1,164 @@
+/*
+ *
+ * @group withdrawal-rolldown
+ */
+
+import { ApiPromise } from "@polkadot/api";
+import { getApi, initApi } from "../../utils/api";
+import { ETH_ASSET_ID, GASP_ASSET_ID } from "../../utils/Constants";
+import { setupApi, setupUsers } from "../../utils/setup";
+import { User } from "../../utils/User";
+import { Sudo } from "../../utils/sudo";
+import { Assets } from "../../utils/Assets";
+import { BN_TEN, BN_THOUSAND, signTx } from "gasp-sdk";
+import { Withdraw } from "../../utils/rolldown";
+import { BN, BN_MILLION } from "@polkadot/util";
+import {
+  ExtrinsicResult,
+  filterZeroEventData,
+} from "../../utils/eventListeners";
+import { getEventResultFromMangataTx } from "../../utils/txHandler";
+
+let api: ApiPromise;
+let testUser: User;
+let gaspIdL1Asset: any;
+let withdrawalAmount: BN;
+let treasuryAccount: string;
+let ethIdL1Asset: any;
+let DEFAULT_AMOUNT: any;
+let chain: any;
+
+beforeAll(async () => {
+  try {
+    getApi();
+  } catch (e) {
+    await initApi();
+  }
+  await setupApi();
+  api = getApi();
+  gaspIdL1Asset = JSON.parse(
+    JSON.stringify(await api.query.assetRegistry.idToL1Asset(GASP_ASSET_ID)),
+  );
+  ethIdL1Asset = JSON.parse(
+    JSON.stringify(await api.query.assetRegistry.idToL1Asset(ETH_ASSET_ID)),
+  );
+  withdrawalAmount = BN_MILLION;
+  treasuryAccount = "0x6d6f646c70792f74727372790000000000000000";
+  DEFAULT_AMOUNT = BN_THOUSAND.mul(BN_TEN.pow(new BN(18)));
+  chain = "Ethereum";
```

Comment:
declare those as constants instead of "let"

Reply:
define them on the test file, replacing those "let" declarations. If the values do not change, do not see the reason why having them assigned here.

Reply:
Done